### PR TITLE
feat: Implement Contribute service page

### PR DESCRIPTION
**Pull request by mohandast52**
- [Contribute page](https://olas-website-git-mohan-contribute-service-autonolas.vercel.app/contribute#what-is-olas-contribute-service)

https://github.com/user-attachments/assets/f0b44476-99ee-4084-809c-bf817e861c44

### CODE REVIEW

File: pages/contribute.jsx [hunk 1]
```diff
@@ -1,13 +1,225 @@
...
+          Track community impact, transform your network
+        </h2>
+        <div className="md:hidden mb-8">
+          <HeroImage />
+        </div>
+        <div className={TEXT_SMALL_CLASS}>
+          Leverage autonomous services to track and enhance the influence of
+          your community.
+        </div>
+        <Button variant="default" size="xl" className="mt-6 w-full md:w-auto">
```
Comment (Tanya-atatakai) [inline pages/contribute.jsx:42]: can we pass `asChild` prop to the button? so the whole buttons is a link, but not only the text

File: pages/contribute.jsx [hunk 2]
```diff
@@ -1,13 +1,225 @@
...
+              key={index}
+              className="flex rounded-lg border border-purple-200"
+            >
+              <p
+                className="flex items-center text-5xl p-4  font-bold text-purple-500 border-x border-l-0 border-purple-200 rounded-l-lg"
+                style={{
+                  background: 'linear-gradient(180deg, #FFF 0%, #FAF0FF 100%)',
+                  // color: 'white',
+                  /* eslint-disable-next-line max-len */
+                  // textShadow: '-1px -1px 0 #a855f7, 1px -1px 0 #a855f7, -1px 1px 0 #a855f7, 1px 1px 0 #a855f7',
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply pages/contribute.jsx:143]: works fine for me, can you elaborate please? and can we remove the code if it's not working?

image

File: pages/contribute.jsx [hunk 3]
```diff
@@ -1,13 +1,225 @@
...
+            alt="OLAS Utility"
+            src="/images/contribute-page/cta-background.png"
+            width={316}
+            height={294}
+          />
+          <Button
+            variant="default"
+            size="lg"
+            className="absolute w-[200px] top-[132px] left-1/2 -translate-x-1/2"
+            onClick={() => window.open(
```
Comment (Tanya-atatakai) [inline pages/contribute.jsx:203]: can we use anchor tag instead?

File: pages/contribute.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply pages/contribute.jsx]: ahh see, sharp eye 👍

**Review by Tanya-atatakai — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Review by Tanya-atatakai — CHANGES_REQUESTED** [review]
just realised this page should point to the `/services/contribute` and the old one can be removed

**Comment by Tanya-atatakai** [comment]
ah sorry, ok

**Review by Tanya-atatakai — APPROVED** [review]

**Review by truemiller — APPROVED** [review]

**Pull request merged**